src/Ideas.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 
-import './Ideas.css';
+import './Ideas.scss';
```

Comment:
I think this needs to import the CSS file, which is generated by SCSS? We don't use it either way, so maybe not a big deal.